tests/general/LineCountBearTest.py

```diff
@@ -40,3 +42,52 @@ def test_exclude_blank_lines(self):
                                 severity=RESULT_SEVERITY.NORMAL,
                                 file='default')],
             filename='default')
+
+    def test_of_range_allowed_for_lines_per_file(self):
```

Comment:
IMO you should drop the `of` from the function name and rename it to `test_range_allowed_for_lines_per_file`
`test_of` didn't sound like a test function name while I was reading it.